### PR TITLE
Update network-config.yaml

### PR DESCRIPTION
**Pull request by rbenamin**
cronos: host adresse war falsch seitens BP angegeben, mit dieser config gehts bei mir

### CODE REVIEW

File: network-config.yaml [hunk 1]
```diff
@@ -1,48 +1,332 @@
...
+        id: optimism-test
+        host: https://kovan.optimism.io
+        explorer: https://api-kovan-optimistic.etherscan.io/api
+        multicall2: "0x2DC0E2aa608532Da689e89e237dF582B783E552C"
+  - name: Polygon
+    networks:
+      - name: Mainnet (Infura)
+        chainid: 137
+        id: polygon-main
+        host: https://polygon-mainnet.infura.io/v3/68be3027359a4525a4414220492d931c
```
Comment (kurzi2704) [inline network-config.yaml]: Could you please remove your Infura ID ?

**Review by kurzi2704 — CHANGES_REQUESTED** [review]
Looks good to me except that you have added your infura ID, which I do not really want to add to the repository

**Comment by rbenamin** [comment]
i will delete it, of course you are right

File: network-config.yaml [hunk 2]
```diff
@@ -1,48 +1,332 @@
...
+        explorer: https://api.polygonscan.com/api
+        multicall2: "0xc8E51042792d7405184DfCa245F2d27B94D013b6"
+      - name: Mumbai Testnet (Infura)
+        chainid: 80001
+        id: polygon-test
+        host: https://polygon-mumbai.infura.io/v3/...... # < update here your projectID from infura
```
Comment (kurzi2704) [inline network-config.yaml]: https://polygon-mumbai.infura.io/v3/$WEB3_INFURA_PROJECT_ID 

then we can use an environment variable for that

**Review by kurzi2704 — CHANGES_REQUESTED** [review]
$WEB3_INFURA_PROJECT_ID would be better than just dots, because we can then use an environment variable

**Comment by kurzi2704** [comment]
Update of networks list - thx @rbenamin

**Pull request merged**